File: P4_EC_Sergio_Aísa/Factorial.c
#include <stdio.h>
int factorial(int num);
int main () {
	printf("Introduzca el numero cuyo factorial quiera calcular: \n");
	int num, resultado;
	scanf("%d", &num);
	resultado = factorial(num);
	if(resultado == -1){
		printf("Introduzca un numero positivo porfavor.\n");
	}else{
		printf("Resultado: %d", resultado);
	}

}
int factorial(int num){
	int total = 1;
	
		if(num < 0){
			return -1;
		}
		else{
			for(int i = num; i > 0; i--){
				total = total * i;
			}
		}
			return total;
		}
		
		

File: P6_EC_Sergio_Aisa/factorial.py
#Realice un programa que lea un entero n y llame a la función factorial
#y devuelve el factorial del número que se le pasa como argumento. Hay que
#imprimir el resultado por pantalla

def factorial(n):
	output = 1
	for i in range (n, 0, -1):
		output *= i
	return output

n = int(input("Introduzca el numero cuyo factorial quiera calcular\n"))
output = factorial(n)
print("Resultado: ", output)
